Hunting Queries/Corelight/CorelightExternalServices.yaml

```diff
@@ -0,0 +1,27 @@
+id: 3e42a796-9a4c-4ebf-a0e0-5282947986b5
+name: Corelight - External Facing Services
+description: |
+  'Query searches for external facing services.'
+severity: Low
+requiredDataConnectors:
+  - connectorId: Corelight
+    dataTypes:
+      - Corelight_CL

```

Comment:
If we are using parser in query datatype should be parser name. So could you please update the datatype name to Corelight.

Reply:
updated